Makefile

```diff
@@ -58,7 +61,7 @@ install-vendor-dev:
 # cleaning rules
 ###
 clean: clean-dist clean-test clean-vendor
-	test ! -e $(DIST).zip || rm $(DIST).zip
+	rm -f $(DIST_PREFIX)*.zip
 
 clean-dist:
 	test ! -d .$(DIST)/ || rm -r .$(DIST)/
```

Comment:
This target generate warning when not ran from a git repo, but an extracted distfile.

```[9:53:55] @ombp2 ~/learnosity_sdk-0.9.1-1-g878e084$ make clean                                                                                                                              
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
test ! -d .learnosity_sdk-/ || rm -r .learnosity_sdk-/
test ! -d learnosity_sdk-/ || rm -r learnosity_sdk-/
rm -f src/tests/junit.xml
test ! -d vendor || rm -r vendor
rm -f learnosity_sdk*.zip
```

I'm starting to think that maybe the easiest would be not to include the Makefile at all in the DIST, as it only enables use cases we don't want to support.

Reply:
Yeah, I agree. I'll fix up the PR to do that.